cmd/root.go

```diff
@@ -47,9 +48,14 @@ func initConfig() {
 		}
 	}
 
+	if !configFileUsed {
+		log.Info("config file not found, proceeding without it")
+	}
+
```

Comment:
I think the placement of this if statement introduces a bug caused by the logger formatter not being configured until later (regression, we've seen this one before IIRC).